UPGRADE-15.0.md

```diff
@@ -861,6 +861,10 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   ParameterColorValueDataFixture is added to handle assigning hex values to color parameters
 -   see #project-base-diff to update your project
 
+#### remove product stock from calculated sale exclusion calculation ([#3159](https://github.com/shopsys/shopsys/pull/3159))
```

Comment:
please, add an entry to the storefront part as well, there should be something like "all the add to cart buttons are now disabled when product quantity is zero or negative"

Reply:
Done.

I like the outcome very much as well. Makes so much more sense than the first try ☺️